src/utilities/HYPRE_utilities.h

```diff
@@ -61,6 +61,7 @@ typedef int HYPRE_Int;
 typedef float HYPRE_Real;
 #define HYPRE_REAL_MAX FLT_MAX
 #define HYPRE_REAL_MIN FLT_MIN
+#define HYPRE_TRUE_MIN FLT_TRUE_MIN
```

Comment:
Should this be called `HYPRE_REAL_TRUE_MIN` for consistency with the above defines? Same for the other definitions below

Reply:
Yes, this was a minor comment

Reply:
Even though I tend to like short names as well (a carryover from my fortran days when 8 characters was the limit :), I vote for inserting the 'REAL_' prefix, since it matches the C naming convention more closely.  2 cents.